Latency.js

```diff
@@ -40,6 +63,37 @@ module.exports = function(callback) {
 
 }
 
+function filterLatencies(query, filter_object) {
+
+  var result = GLOBAL_LATENCIES;
+
+  if (filter_object['network']) {
+    result = result.filter(function(x) {return eval(createCommandString(query, filter_object, 'network'))})
```

Comment:
Gebruik nooit `eval`, en helemaal niet in combinatie met user input. Als je dan een foutje maakt in de input sanitization kunnen gebruikers willekeurige code uitvoeren op de server.

Ik zou zoiets doen:

1. requestedNetworks = query["network"].split(",");
2. filter(function(x) { return requestedNetworks.indexOf(x) !== -1 })

En hetzelfde voor station, location, channel